duniverse.opam

```diff
@@ -11,6 +11,7 @@ depends: [
   "ocaml" {>= "4.05.0"} 
   "dune" {build >= "1.2.0"}
   "ppx_sexp_conv"
+  "ppx_deriving"
```

Comment:
Is the end game still merging this into `dune`? If that's the case I don't think it's a good idea to add this dependency.

Reply:
Does `stdune` have some helpers for that? It's a bit of a hassle to write those by hand in the meantime.

Reply:
Not really. There are some extra types like `Ordering.t`, but the comparisons are written by hand. It's true that it's a lot of boilerplate to add, but either you'll have to do it now or later. So it's really your call :)